pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -259,7 +259,8 @@ func newNetworkAddonsForCR(cr *hcov1alpha1.HyperConverged) *networkaddons.Networ
 		Spec: networkaddons.NetworkAddonsConfigSpec{
 			Multus:          &networkaddons.Multus{},
 			LinuxBridge:     &networkaddons.LinuxBridge{},
-			ImagePullPolicy: NetworkAddonsImagePullPolicy,
+			KubeMacPool:     &networkaddons.KubeMacPool{},
+			ImagePullPolicy: v1.PullPolicy(NetworkAddonsImagePullPolicy),
```

Comment:
What about sr-iov?

Reply:
Not fully integrated in 0.3.0. Will be added with 0.4.0.

Reply:
I know, but I know @dankenigsberg want's it in anyway.

Reply:
**Edit** it won't be added in 0.4.0. We want to test it, however, it won't be enabled in HCO. If you want to try it, you have to edit NetworkAddonsConfig.Spec.Sriov directly.

Reply:
OK